package.json

```diff
@@ -56,14 +56,15 @@
     "@message-queue-toolkit/schemas": "^4.0.0",
     "@message-queue-toolkit/sns": "^18.0.1",
     "@message-queue-toolkit/sqs": "^17.2.0",
-    "@prisma/client": "^5.22.0",
-    "@prisma/instrumentation": "^5.22.0",
     "@scalar/fastify-api-reference": "^1.25.61",
     "@supercharge/promise-pool": "^3.2.0",
     "amqplib": "^0.10.4",
     "awilix": "^12.0.3",
     "awilix-manager": "^5.5.0",
     "bullmq": "^5.25.4",
+    "dotenv": "^16.4.5",
```

Comment:
Yeah, we should definitely be able to avoid this. I've followed Drizzle's instructions for installation but something has changed and they are squeezing in `dotenv` in the recommended instructions, but it looks like it's not strictly needed. Maestro doesn't have it and works fine. I'll remove it!